ui/round/src/plugins/keyboardMove.ts

```diff
@@ -67,7 +67,9 @@ lichess.keyboardMove = function (opts: Opts) {
         readClocks(opts.ctrl.clock());
         clear();
       }
-    } else if (submitOpts.yourMove && v.length > 1) {
+    } else if (submitOpts.yourMove && v.length > 0 && legalSans && !sanCandidates(v, legalSans).length) {
+      // submitOpts.yourMove is true only when it is newly the player's turn, not on subsequent
```

Comment:
This is the somewhat odd behavior I found while looking into this issue. I wanted to raise it in a comment in case this was unknown/undesirable